chain/events/filter/index.go

```diff
@@ -480,13 +510,20 @@ func (ei *EventIndex) prefillFilter(ctx context.Context, f *eventFilter, exclude
 		clauses = append(clauses, "event.tipset_key_cid=?")
 		values = append(values, f.tipsetCid.Bytes())
 	} else {
-		if f.minHeight >= 0 {
-			clauses = append(clauses, "event.height>=?")
+		if f.minHeight >= 0 && f.minHeight == f.maxHeight {
+			clauses = append(clauses, "event.height=?")
 			values = append(values, f.minHeight)
-		}
-		if f.maxHeight >= 0 {
-			clauses = append(clauses, "event.height<=?")
-			values = append(values, f.maxHeight)
+		} else {
+			if f.maxHeight >= 0 && f.minHeight >= 0 {
+				clauses = append(clauses, "event.height BETWEEN ? AND ?")
```

Comment:
Just out of curiosity: Is there a difference between using `>= and <=` AND `BETWEEN` ?

Reply:
I don't think so, I couldn't observe a measurable speed difference when I was trying this, it ended up _feeling_ better to specify it this way to be more explicit (you're welcome to object to an aesthetic change if you disagree).

Reply:
No was just curious. This is lgtm.